src/core/utils/useragent.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Returns whether the current browser is Microsoft Edge.
+ * Tries to use User-Agent clients hints, and defaults to
+ * using the User-Agent string if clients hints are not supported.
+ *
+ * @returns {boolean}
+ */
+export function isMicrosoftEdge () {
+  const brands = navigator.userAgentData?.brands;
+  if (brands && brands.length > 0) {
+    return !!brands.find(b => b.brand === 'Microsoft Edge');
+  } else if (navigator.userAgent) {
+    return navigator.userAgent.includes('Edg');
```

Comment:
Do you think just checking these 3 characters might lead to some false positives? Is it practical to instead list out the possible full strings?

Reply:
I spent quite a while trying to find better ways, but this is the recommended way for checking for Edge. I agree it's strange because any other word that starts with Edg will give us a false positive. I don't think it's practical to list out all possible strings.

Reply:
On the bright side it looks like the future is User-Agent Client Hints!